src/spring/azext_spring/custom.py

```diff
@@ -21,7 +21,7 @@
 from azure.mgmt.core.tools import (parse_resource_id, is_valid_resource_id)
 from ._utils import (get_portal_uri, get_spring_sku, get_proxy_api_endpoint, BearerAuth)
 from knack.util import CLIError
-from .vendored_sdks.appplatform.v2023_03_01_preview import models, AppPlatformManagementClient
+from .vendored_sdks.appplatform.v2023_05_01_preview import models, AppPlatformManagementClient
```

Comment:
It is suggested to import specific model rather all models.

Reply:
Should be the same version